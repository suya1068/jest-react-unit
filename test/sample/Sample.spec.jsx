import React from "react";
import Sample from "src/sample/Sample";

describe("<Sample />", () => {
    it("renders three <Sample /> components", () => {
        const tree = renderer
            .create(<Sample />)
            .toJSON();
            
        expect(tree).toMatchSnapshot();
    });
});