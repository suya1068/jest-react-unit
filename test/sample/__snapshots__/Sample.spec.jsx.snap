// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sample /> renders three <Sample /> components 1`] = `
<h1>
  Hi, Jest!
</h1>
`;
